common/views/components/ApiToolbar/ApiToolbar.tsx

```diff
@@ -11,6 +11,7 @@ type Prop = {
   label: string;
   value?: string;
   link?: string;
+  displayLink?: boolean;
```

Comment:
Rather than using a boolean variable here, let's lean on the type system and return `undefined` instead of a partially-populated `Props` object.